src/pages/index.jsx

```diff
@@ -26,9 +26,9 @@ export default function Home() {
           <Speakers />
           <Schedule />
           <Papers />
+          <Team />
           <Sponsors />
           <Organization />
-          <Team />
```

Comment:
I honestly prefer the team where it was :sweat_smile:

Reply:
Me too :point_up:

Reply:
I don't prefer it